core/src/blocktree_processor.rs

```diff
@@ -16,16 +18,15 @@ use std::result;
 use std::sync::Arc;
 use std::time::{Duration, Instant};
 
-use rand::seq::SliceRandom;
-use rand::thread_rng;
-
 pub const NUM_THREADS: u32 = 10;
+use solana_rayon_threadlimit::thread_count::get_thread_count;
```

Comment:
nit: drop the thread_count module?  `use solana_rayon_threadlimit::get_thread_count;`